examples/srd_v1_3x2.yaml

```diff
@@ -4,37 +4,38 @@ generate:
     # will once be part of TJPCov interface
     fsky : &fsky 0.35
     add_noise: &add_noise False # add a realization of noise?
-      
+    kmax: &kmax 0.3 # kmax in Mpc^-1 for 2pt analyses
+
   parameters: &parameters
     Omega_k: 0.0
     Omega_b: 0.0492
     Omega_c: 0.26639999999999997  # = 0.3156 - 0.0492
     w0: -1.0
     wa: 0.0
     h: 0.6727
-    sigma8: 0.83 
+    sigma8: 0.83
     n_s: 0.9645
-  
+    transfer_function: 'eisenstein_hu'
     # IA model
     eta_ia: 0.0
     eta_ia_highz: 0.0
     beta_ia: 1.0
     A0: 5.0
-  
+
     mult_bias: 0.0
-  
+
     # magic numbers from Tim
     # for the life of me, I cannot figure out how to get CCL to compute the
     # SRD bias values. So they are hard coded here.
```

Comment:
Ok, these biases should be 0.95/growth. Can you check that they are indeed this? And there should be a nice way for augur to calculate those itself. Perhaps similar to how I enabled np.logspace to be used for ell edges?

Reply:
I can help with this, because I remember banging my head against a wall trying to figure it out a couple years ago - they aren't 0.95 / growth. There was a mistake in the calculation of the bias values that got used in the SRD, something to do with normalisation, which is why these are "magic numbers from Tim" i.e. the values with this mistake, which you have to use if you want to match the SRD. In general, when we are not trying to match the SRD, calculating 0.95 / growth makes sense to me.

Reply:
Ok, it would be good to see how big the difference this is. And that what you are describing doesn't get lost in time. Basically, when we update SRD and the numbers are different, we can say: they are this different due to different bias assumptions, and this different due to better modelling of whatever, etc

Reply:
Is there a good place to make note of this so it's not lost in time?